tests/integration/tc_71_10.c

```diff
@@ -28,9 +29,11 @@ int main() {
     send_run_mode(SHEPHERD, TELEOP);
 
     // Start the timer and press A
-    int32_t start = millis() % 1000000000;  // 9 digits, just like TimeTestDevice
-    buttons |= get_button_bit("button_a");
+
+    uint64_t start = millis();
+    buttons |= (1 << BUTTON_A);
```

Comment:
`buttons |= get_button_bit("button_a");` should be ok. Don't break abstraction barrier

Reply:
Oh whoops yea when resolving a merge conflict I probably accepted the wrong one. Let me fix that